### DISCUSSION TITLE
[Boxplot] Add a vertical axis slider

### DISCUSSION THREAD

**Discussion by ValentinC-BR** [apache/superset#18507] [Ideas]
**Is your feature request related to a problem? Please describe.**
The boxplot currently needs a user-defined axis range (see this feature request : https://github.com/apache/superset/issues/16745 )

However, this will only partially solve the problem. For better data exploration, providing the user with a slider to manually zoom in the y-axis would be a great enhancement.

<br>

**Describe the solution you'd like**
Add an option in the "Customize" panel : 
![image](https://user-images.githubusercontent.com/79460908/133972171-706bce63-66fd-4e02-a85f-c04ff0335acd.png)

When set to True, this display a vertical range slider (next to the y-axis) like this : 
![image](https://user-images.githubusercontent.com/79460908/133971704-426777bc-b3ec-4adc-b314-5dec0087d9a2.png)

<br>

**Describe alternatives you've considered**
Range sliders like the x-axis zoom options in line charts can also be a solution, but would take too much space :
![image](https://user-images.githubusercontent.com/79460908/133972041-b5da86e2-9d0d-4351-ac3e-d5ab6bf7f3b5.png)

**Additional context**
/

**Comment by junlincc** [comment]
@ValentinC-BR thanks for the suggestion! looks like you have the feature working in your production already? we would love to hear how it benefit your org and discuss how to make it work in Superset :)

**Comment by ValentinC-BR** [comment]
Hello,
I just took screenshots of others charts and/or tools, it is not implemented in our Superset.

We're actually using the master branch (only the major releases, when they're stable) without any add-ons or modifications.

**Comment by sfirke** [comment]
I just got a request for this from one of my users - they want to see outliers in the boxplot, but then be able to zoom in past them to investigate the majority of the data.
